charts/mlrun-ce/values.yaml

```diff
@@ -147,7 +147,10 @@ mlrun:
     mode: full
 
 jupyterNotebook:
-  awsInstall: false
+  serviceAccount:
+    create: true
```

Comment:
why do you need the create?

Reply:
in oreder ro be able run kubectl from pod